### PR TITLE
feat: allow to include the navigation bar in a single line header

### PR DESCRIPTION
**Pull request by yopemu**
Allow the option to include the navigation bar in a single line as a condensed header. Instead of 2 lines only one (the top one) will contains the logo, title and navigation bar, as well as the settings. As discussed with Debray Alize and Gfeller Philipp.

Team NES
Langhard Matthias
Pedroso Yohandris

### CODE REVIEW

**Comment by yopemu** [comment]
Hi @gfellerph as suggested, here is the direct PR

**Comment by gfellerph** [comment]
Resolution pattern discussion: thumbs up

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html
Comment (gfellerph) [inline packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html:120]: This causes screenreaders to announce "navigation navigation" as the `<nav>` already prompts a (localized) announcement of navigation. When adding this statically, there is also no chance to localize it.

```suggestion
```

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html
Comment (gfellerph) [inline packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html:5]: Why was this id added? There is a chance of this id already being in use by a project, which could cause hard to find errors. If this id is necessary, we should prefix it with `post-` or an even more specific prefix. Otherwise, please remove it.

*Same goes for the other instances where ids were added.*

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html
Comment (gfellerph) [inline packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html:20]: This causes screenreaders to announce "navigation navigation" as the `<nav>` already prompts a (localized) announcement of navigation. When adding this statically, there is also no chance to localize it.

```suggestion
```

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts
Comment (gfellerph) [inline packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts:364]: Nice!

File: packages/styles/src/components/intranet-header/_top-navigation.scss
Comment (gfellerph) [inline packages/styles/src/components/intranet-header/_top-navigation.scss:1]: There's a one pixel gap around the logo, possibly caused by the nav items that are 2px bigger than the logo.

image

**Review by gfellerph — CHANGES_REQUESTED** [review]
Thanks for doing the extensive rework necessary for providing this option and also thinking about documentation. The PR generally looks good, there are some things however that need to be fixed before we can merge it.

**Comment by gfellerph** [comment]
Notice for @swisspost/design-system-maintainers, this PR needs to go to main and then needs a cherry pick to `release/v8`.

**Comment by yopemu** [comment]
@gfellerph Thank you for the review and your feedback. I will have a look at your suggestions.

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html
Comment (yopemu) [reply packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html:120]: I see. I only added it afterwards because sonar failed when it wasn't there. I will remove it again.

File: packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html
Comment (yopemu) [reply packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html:5]: Yes, the IDs are needed to identify the containers and then calculate the available space. This identifies them uniquely. I see the point with the other projects and will add the prefix “post-”.

**Comment by riscie** [comment]
Any news on this one @gfellerph? Is the ball in our court?

**Comment by gfellerph** [comment]
Superseded by #4084 because this change here was based on main, where the Demo App is already removed and the Intranet Header will be removed soon (to be replaced by the new web component header).

**Pull request closed without merging**
